openhtf/output/web_gui/src/app/station.service.ts

```diff
@@ -98,7 +102,9 @@ export class StationService extends SubscriptionService {
   onmessage(msg) {
     this.reachable = true;
     let parsedData = JSON.parse(msg.data);
-    let test_uid = parsedData.test_uid, state = parsedData.state;
+    let test_uid = parsedData.test_uid;
+    let state = parsedData.state;
+    state.test_uid = test_uid  // Add test_uid to the test state.
```

Comment:
I've added a commit `Keep test UID separate from state` Let me know what you think!